lighthouse-core/scripts/i18n/collection-util.js

```diff
@@ -239,8 +239,43 @@ function createPsuedoLocaleStrings(strings) {
   return psuedoLocalizedStrings;
 }
 
+/**
+ * @param {Record<string, ICUMessageDefn>} messages
+ * @returns {Record<string, ICUMessageDefn>}
+ */
+function bakePlaceholders(messages) {
```

Comment:
`preprocessPlaceholders`?

Reply:
I like bake 🍰

Reply:
add comment saying "this preprocesses messages for markdown links and generates placeholders for them" or something

Reply:
general comments linking the functions to the steps in the doc would be awesome

no necesssarily re explaining, but just so you know what step you're looking at.